pom.xml

```
@@ -123,6 +123,12 @@
             <artifactId>janino</artifactId>
             <version>3.1.12</version>
         </dependency>
+        <dependency>
+            <!-- Upgrade transitive dependency of org.apache.spark:spark-network-common_2.13 to fix CVE-2024-7254 -->
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>3.25.5</version>
+        </dependency>
```

```suggestion
            <version>3.25.5</version>
            <scope>provided</scope>
        </dependency>
```

You might want to check if other dependencies also have the wrong scope.